### DISCUSSION TITLE
Is it possible to delete a job with a delay?

### DISCUSSION THREAD

**Discussion by chuck6a6ka** [taskforcesh/bullmq#1444] [Q&A] [answered]
I can’t find an answer to the question whether it is possible to delete a task with a delay by ID.

_Situation._
At what point in time I created the task with a delay. After some amount of time, I decided that I no longer need this task and I want to delete it myself. In the documentation, I just found that the "Job" class has a "remove" method that deletes job, but I will no longer have access to this class after some time

**Comment by manast** [comment, marked as answer]
If you know the id of the job just instantiate the job (https://api.docs.bullmq.io/classes/Job.html#fromId) and then remove it (https://api.docs.bullmq.io/classes/Job.html#remove).
